src/main/java/com/minecolonies/entity/ai/citizen/miner/EntityAIStructureMiner.java

```diff
@@ -287,8 +286,13 @@ else if (ladderOrientation == EnumFacing.NORTH)
         {
             throw new IllegalStateException("Ladder metadata was " + ladderOrientation);
         }
+
+        int x = buildingMiner.getLadderLocation().getX();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'x' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)